recipes/medical_imaging.ipynb

```diff
@@ -106,12 +107,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# !pip install kili # uncomment if you don't have kili installed already\n",
+    "!pip install --upgrade kili # uncomment if you don't have kili installed already\n",
```

Comment:
"# uncomment if you don't have kili installed already\n" id not legit anymore since you removed the "#"

Reply:
Done. Pls verify